### PR TITLE
Prove exact Apertus GGUF runtime support

### PR DESCRIPTION
**Pull request by justinchuby**
## Summary

- Promote only the immutable Apertus v1.1 1.5B Instruct BF16 GGUF exact-float CPU route to final-package runtime-supported status.
- Accept pinned llama.cpp v1.1 serialization: unqualified `xielu.*` metadata and factorless default RoPE, while rejecting conflicting xIELU keys, incomplete/malformed factors, unsupported scaling metadata, ignored `original_context_length`, and lossy factor storage.
- Preserve graph identity independently from the subsequently written `export_report.json`, then validate the complete final runtime package including that report.
- Auto-select exact immutable runtime evidence without requiring callers to repeat its evidence-pinned tokenizer repository/revision. Matching explicit flags remain accepted; conflicting explicit identities fail before serialization.
- Reconcile #697 tokenizer policy without adding a broad Tekken promotion: unmatched artifacts remain deferred and tokenizer-omitted.
- Replace a build-time deferred tokenizer report with supported/exported after exact materialization, so the validated package is truthfully complete and end-to-end runnable.
- Use an independent raw `gguf.GGUFReader` + `dequantize` oracle with Mobius loading, mapping, tensor-processing, and normalization helpers patched to raise.

## Immutable evidence

- GGUF: `MrMeOrYou/Apertus-v1.1-1.5B-Instruct-GGUF@88c75ad49566d3c2157d03709bf772262c3241ed`
- File: `Apertus-v1.1-1.5B-Instruct-BF16.gguf`
- Size/SHA-256: `3,028,052,608` / `f9ec154d0ec29dad1f6465b458b7f27bd25ad7b9a3899233ae98ca6d358501c2`
- Official config/tokenizer: `swiss-ai/Apertus-v1.1-1.5B-Instruct@9e9d01154446a645d30f04174cf1515a38058be7`
- Tensor census: 163 tensors (`BF16`: 98, `F32`: 65)
- Graph fingerprint: `4bb91bade19d41559cb524e28692453851fe67274cc58109787ee968df3e0fe5`
- Final runtime-package fingerprint: `97582549e9c5b4114f3bcfa81c92f16aba9d14dd0ea0d2b20acaefd9060e6486`

The real-artifact test invokes runtime packaging without tokenizer source flags and proves automatic evidence-pinned materialization, full-logit prefill, fresh and cached decode parity, full-sequence replay, rollback, batch reorder, 20 exact greedy decode steps, deterministic ORT GenAI generation, tokenizer vocabulary/special-token/chat-template identity, graph/config/package fingerprints, package/report roundtrip, complete validated export reporting, and fail-closed artifact/source/tokenizer/runtime mismatches. Observed prefill max absolute difference was `0.00016784668`; cached-vs-fresh differences remained below `4e-5`.

## Truthful support boundary

`source_fidelity=true` because BF16 values are exactly representable after import to float32. `storage_quantized=false` because the emitted ONNX package stores float tensors. Runtime support is evidence-gated to this exact artifact, official tokenizer revision, CPU import route, ONNX Runtime 1.29.0, and ORT GenAI 0.15.2. Other Apertus artifacts retain the generic best-effort advisory behavior and cannot inherit this evidence. This does not claim a generic independent Tekken tokenizer promotion, preserved-quantized runtime, CUDA, or other runtime versions. No Apertus model-ID/revision production gate was added, and the #675 source/import-route identity remains intact.

## Validation

- Flag-free immutable Apertus runtime probe — 1 passed, 13 deselected; an initial successful run hit an intermittent ORT GenAI process-exit mutex error, and the clean replay passed with exit 0
- Focused Apertus/runtime-evidence/runtime-package/tokenizer/docs/census/schema tests — 436 passed
- `python -m pytest tests/build_graph_test.py tests/cli_test.py src/ -q -k 'not phi4mm and not apply_weights_unknown' --tb=short` — 9,675 passed, 64 skipped, 12 deselected, 1 subtest passed
- Exact evidence unit coverage: no tokenizer flags succeeds; matching explicit flags succeeds; wrong repository/revision conflicts before save; unmatched artifact does not auto-materialize
- `lintrunner f --output oneline --all-files && lintrunner -a` — clean
- `python scripts/generate_gguf_support_docs.py --check` — current
- Focused mypy — clean
- Final independent medium review found one legacy-case assertion-scope issue; it was fixed, and the follow-up review found no issues.
- Both existing GitHub review threads were answered and resolved before the final push.

## Waivers

- Preserved-quantized runtime: not claimed; representative evidence is BF16 imported exactly into float storage.
- CUDA, Foundry Local, and Olive probes: outside this CPU ORT GenAI evidence promotion and not claimed.
- Separate L4/L5 golden files: waived in favor of the immutable real-artifact harness, which asserts full logits and all 20 generated token IDs rather than prefix-only output.

Semantically rebased and validated at exact base `2c981f2b631a907f43c57d3235647c22cc888eae`; exact head `256c2b15f17ec327634ef5cae1d1e902a07f5309`. Do not merge as part of this task.

### CODE REVIEW

File: src/mobius/integrations/gguf/_config_mapping.py [hunk 1]
```diff
@@ -1417,8 +1436,22 @@ def per_layer(suffix: str) -> tuple[float, ...]:
...
+    )
+    scaling_type = metadata.get(f"{arch}.rope.scaling.type")
 
-    if rope_names == {"rope_freqs.weight"}:
+    if not rope_names:
+        if scaling_type not in {None, "", "none"}:
+            raise ValueError(
+                "Apertus factorless RoPE requires rope.scaling.type to be absent or 'none'"
+            )
+        short_factors = long_factors = None
```
[the remark this answers is not in the document]
Comment (justinchuby) [reply src/mobius/integrations/gguf/_config_mapping.py:1482]: No change needed here: `long_factors` is read by the shared validation and `rope_scaling` construction after this branch, so assigning it to `None` is required for definite initialization and for the factorless path to remain explicit. Removing only that assignment would make the downstream read unbound.

File: src/mobius/integrations/gguf/_config_mapping.py [hunk 2]
```diff
@@ -1417,8 +1436,22 @@ def per_layer(suffix: str) -> tuple[float, ...]:
...
+        original_context = config.max_position_embeddings
+    elif rope_names == {"rope_freqs.weight"}:
```
[the remark this answers is not in the document]
Comment (justinchuby) [reply src/mobius/integrations/gguf/_config_mapping.py:1484]: Fixed after rebasing current main. `apertus.rope.scaling.original_context_length` is now rejected unless the complete short/long factor pair is present, with a focused factorless rejection test. This also prevents the single `rope_freqs.weight` form from silently ignoring that metadata.
